### DISCUSSION TITLE
Option to not expand images when enabling panorama mode

### DISCUSSION THREAD

**Discussion by PurpleHel** [AprilSylph/XKit-Rewritten#1145] [Suggestion Box]
I like text taking up the full width, but I don't like images being resized. I'd like an option for panorama mode to leave images at their original size.

**Comment by marcustyphoon** [comment]
Interesting. That... might be possible; I'm not sure!

I won't look into it for a while, almost certainly, since there are a lot of internal changes that have to be made to Panorama that are just sort of on the backlog.
